dev/tests/settings_local.py

```diff
@@ -17,8 +17,8 @@
 }
 
 IDSUBMIT_IDNITS_BINARY = "/usr/local/bin/idnits"
-IDSUBMIT_REPOSITORY_PATH = "test/id/"
-IDSUBMIT_STAGING_PATH = "test/staging/"
+IDSUBMIT_REPOSITORY_PATH = "/assets/ietf-ftp/internet-drafts/"
+IDSUBMIT_STAGING_PATH = "/assets/www6s/staging/"
```

Comment:
in the long run, staging needs to be reworked anyhow to be in db, not on disk. I'd like to squeeze www6s out of the dev container configuration, but until we pull everything out of the proceedings dir (where the meeting materials are currently living), we can keep /assets/www6s to reflect the production reality.